### ISSUE TITLE
How about speeds between conv_TBC of fairseq-py and nn.conv1D in inferencing?

### ISSUE THREAD

**Issue by jhjungCode** [r9y9/deepvoice3_pytorch#22]
In fair team saying,
They said there's **a big speed difference** their own conv_temperal and original nn.conv1d **in inferencing**.

Have you checked the speed of this two modules while removing fairseq-py dependency?

By the way, I agree implementation without dependency. It make me readily to see overall code flow.
good job!

**Comment by r9y9** [comment]
Yes, I actually used ConvTBC implementation in the early. However, I switched off because of the two reasons:

1. ConvTBC doesn't support dilation > 1
2. There's no significant speed difference (as I remember correctly, it's just 1 ~ 1.5x faster than nn.Conv1d with kernel_size=3, 5 or 7)

Check out my old code that supports both ConvTBC and nn.Conv1d (defaults ConvTBC, switched automatically nn.Conv1d if dilation > 1) If you are interested: https://github.com/r9y9/deepvoice3_pytorch/tree/70dc880fae185d96effaee97f0ce55b5c0d13b61.
